hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/AbstractNodeExecutor.java

```diff
@@ -28,10 +28,14 @@
 import java.io.IOException;
 import java.io.OutputStreamWriter;
 import java.io.Writer;
+import java.net.InetAddress;
+import java.net.InetSocketAddress;
+import java.net.Socket;
 import java.nio.charset.Charset;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.TimerTask;
```

Comment:
I already know the push back on this :)  however.... I would to see smaller classes.  This seems like functionality that would be in a NodeHealthChecker class.